index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="./styles.css" rel="stylesheet">
+    <title>AwesomeBooks</title>
+  </head>
+<body>
+<header>
+  <nav class="navigation">
+    <h2 class="logo">Awesome Books</h2>
+    <ul class="navBar">
+      <li><a id="booksBtn" href="#">List</a></li>
+      <li><a id="addbookBtn" href="#">Add NewBook</a></li>
+      <li><a id="contactBtn" href="#">Contact</a></li>
+    </ul>
+  </nav>
+</header>
+
+<main>
+  <div class="clock">
+    <div id="date"></div>
+    <div id="clock"></div>
+  </div>
+
+<div id="container">
+  <h1 class="text-center mt-5">Books Library</h1>
+  <div id="books"></div>
+</div>
+
+<form id="addbook">
+  <h2>Add a New Book</h2>
+  <div>
+    <label for="titulo"></label>
+    <input type="text" id="titulo" placeholder="Title of Book">
+  </div>
+  <div>
+    <label for="autor"></label>
+    <input type="text" id="autor" placeholder="Author of Book">
+  </div>
+  <button id="btn">Add</button>
+</form>
+
+<div id="contact">
+  <h2>Contact Information</h2>
+  <p>Want to connect with us. Then What you are waiting for? <br>Contact us we will get back to you!!</p>
+  <ul>
+    <li><b>Our Email:</b> demo@gmail.com</li>
+    <li><b>Our Contact#:</b> +92 345 4576</li>
+    <li><b>Our Address:</b> SteetName abcg, Street# 123 City aghb</li>
+  </ul>
+</div>
+</main>
+
+<footer>
+  <p>Copyright @2022, <a href="https://github.com/Fatimahub333/Awesome_Books/blob/main/MIT.md"></a>
+  Website for Educational purposes, based on wireframe provided by Microverse.
+  </p>
+</footer>
+
+<script type = "module" src="scripts.js"></script>
+</body>
+</html>
+
+
+
+
+
```

Comment:
- [ ] **Optional:** Kindly remove extra white lines. It is not professional to leave excess white lines beyond the required one for linters.